akka-docs/src/test/scala/docs/stream/operators/sourceorflow/MapError.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators.sourceorflow
+
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.{ Sink, Source }
+
+import scala.concurrent.ExecutionContext
+import scala.util.control.NoStackTrace
+import scala.util.{ Failure, Success }
+
+object MapError extends App {
+
+  implicit val system: ActorSystem = ActorSystem()
+  implicit val ec: ExecutionContext = system.dispatcher
+
+  //#map-error
+  Source(-1 to 1)
+    .map(1 / _)
+    .mapError {
+      case _: ArithmeticException =>
+        throw new UnsupportedOperationException("Divide by Zero Operation is not supported.") with NoStackTrace
```

Comment:
Not a big thing but the partial function is meant to return the `UnsupportedOperationException`, not throw it.

Reply:
Thanks for pointing out.